CMakeLists.txt

```diff
@@ -35,13 +35,15 @@ message(STATUS "VTOOL_VERSION = ${VTOOL_VERSION}")
 DISSECT_VERSION()
 GET_OS_INFO()
 
-if (OS_IS_MACOS)
-    message(STATUS "INCLUDING MACPORTS")
+if (APPLE)
+    message(STATUS "Including Homebrew")
```

Comment:
I'm moderately opposed to moving our build environment on macOS to Homebrew over MacPorts.  However, I will concede that using MacPorts is a big issue with CI since it requires Xcode to be installed and configured correctly prior to running.  It also helps that GitHub Actions' macOS VM supports Homebrew as the package manager of choice (https://github.com/actions/virtual-environments/blob/master/images/macos/macos-10.15-Readme.md).  I find MacPorts to be updated more frequently and has better support for compilers.

Luckily, it looks like Xcode is installed already in that environment and installing MacPorts seems to be pretty much supported already OOTB with the help of this GitHub action in the marketplace: https://github.com/keepitsimpletech/macports-github-actions/blob/master/.github/workflows/macports.yml

Reply:
I was hoping to quickly throw together a solution based on this info. My hopes have been smashed... 

https://github.com/keepitsimpletech/actions/tree/master/setup-macports is now 10 months from its last update. It was originally built for macos 10.14. I'd think the action would have received an update if the author were still using this action, because their builds would be failing.

Because it is built for macos 10.14 and not receiving updates, we'll likely need to take ownership of the action code if we intend to use it. You up for taking ownership of this code?

Reply:
In the meantime I'll look for other options...

What do you think about *including both macports and homebrew support*? I don't think there is any harm in including both, for example:
```
include_directories(/opt/local/include)
link_directories(/opt/local/lib)
include_directories(/usr/local/include)
link_directories(/usr/local/lib)
```

Then we get the best of both worlds while not hindering development.

Reply:
Can we build our own MacPorts target similar to this: https://github.com/keepitsimpletech/actions/compare/master...medengineer:master

Reply:
I'd like to burn a bit longer on the macports support for the immediate term today.  I'm not opposed to us maintaining an updated fork of the code provided by `keepitsimpletech/actions`

Reply:
Ok, we'll continue with it. Based on that comparison you showed, it looks like they may have simply moved development to the fork. I had glanced at the fork, but didn't see the adjustment.

I'll contact the author (`medengineer`) about 1) tagging the project 2) outputting the macports version for caching purposes 3) and offering caching instructions in return.

Reply:
@mmulich I've added an updated MacPorts action to `wbia-pypkg-build` that can be seen here: https://github.com/WildbookOrg/wbia-pypkg-build/actions/runs/146632922

Reply:
That could work. Though it's nice as aesthetically pleasing as calling action code. You could probably see it as the difference between calling a function on a line vs doing the logic of the function on a line.

Either way, one would likely want to cache the results of the directory. Either direction will work for me, but I'd somewhat favor the action (i.e. medengineer's macports-setup) for clarity and brevity.

Reply:
I agree, this provides two options for getting MacPorts up and running.  Here are the options I can see: 
1. have the `wbia-pypkg-build` repo build an artifact to load `/opt/local/` onto the build instance as a pre-computed cache
2. Use the action provieded by ` WildbookOrg/wbia-pypkg-build/actions/macports` to get MacPorts installed on the build environment.  This is actually working here (https://github.com/WildbookOrg/wbia-vtool/blob/automated-wheel-w-macports/.github/workflows/python-publish.yml#L37) on the `automated-wheel-w-macports` branch, which is building OpenCV without issue against MacPorts... but is failing to build vtool for some reason.